core/nuxeo-operation.js

```diff
@@ -330,27 +396,82 @@ import './nuxeo-connection.js';
         });
     }
 
+    /**
+     * Handler to abort bulk operations executed through the BAF
+     */
+    _abort(commandId) {
+      // if the operation is aborted, then on next poll we get the correct state
+      return this.$.nx.request().then((request) => {
+        return request
+          .path(`bulk/${commandId}/abort`)
+          .execute({ method: 'put' })
+          .then((status) => {
+            if (this._isAborted(status)) {
+              this.dispatchEvent(
+                new CustomEvent('poll-aborted', {
+                  bubbles: true,
+                  composed: true,
+                  detail: status,
+                }),
+              );
+            }
+          })
+          .catch((error) => {
+            this.dispatchEvent(
+              new CustomEvent('poll-error', {
+                bubbles: true,
+                composed: true,
+                detail: error,
+              }),
+            );
+            throw error;
+          });
+      });
+    }
+
     _isRunning(status) {
       if (status['entity-type'] === 'bulkStatus') {
-        const { state } = status.value;
+        const { state } = status.value ? status.value : status;
```

Comment:
nit: could be made shorter:
```suggestion
        const { state } = status.value || status;
```

Reply:
Thanks 👍